internal/cmd/internal/olmv1/operator_update.go

```diff
@@ -0,0 +1,40 @@
+package olmv1
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/operator-framework/kubectl-operator/internal/cmd/internal/log"
+	v1action "github.com/operator-framework/kubectl-operator/internal/pkg/v1/action"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+)
+
+// NewOperatorUpdateCmd allows updating a selected operator
+func NewOperatorUpdateCmd(cfg *action.Configuration) *cobra.Command {
+	i := v1action.NewOperatorUpdate(cfg)
+	i.Logf = log.Printf
+
+	cmd := &cobra.Command{
+		Use:   "operator <operator>",
+		Short: "Update an operator",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			i.Package = args[0]
+			_, err := i.Run(cmd.Context())
+			if err != nil {
+				log.Fatalf("failed to update operator: %v", err)
+			}
+			log.Printf("operator %q updated", i.Package)
+		},
+	}
+	bindOperatorUpdateFlags(cmd.Flags(), i)
+
+	return cmd
+}
+
+func bindOperatorUpdateFlags(fs *pflag.FlagSet, i *v1action.OperatorUpdate) {
+	fs.StringVar(&i.Version, "version", "", "desired operator version (single or range) in semver format. AND operation with channels")
```

Comment:
```suggestion
	fs.StringVar(&i.Version, "version", "", "Desired operator version (single value or range) in SemVer format. Combined with 'channels' using AND logic.")
```

I do not understand the channels part. What about adding examples using cobra?

Reply:
We need to explicitly state that the default behavior of this option is to check all channels, right?

Reply:
channels part are 1:1 channels you can see on our CRD - it has a good documentation, which I think should be sufficient if you want to know more

resolving following: https://github.com/operator-framework/kubectl-operator/pull/225#issuecomment-2706623506

Reply:
Can you please improve the text?
"AND operation with channels" sounds a bit unclear—how do tools usually handle this? It feels a little odd.
It’s not very clear for the end user, and good UX means providing helpful and intuitive explanations in the tool.

Reply:
`AND operation with ` for the flag description seems not offend used right?
But that is a NIT for sure